lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -349,10 +349,10 @@ class CategoryRenderer {
       percentageEl.title = Util.i18n.strings.errorLabel;
     }
 
-    if (this.hasApplicableAudits(category)) {
+    // Render a numerical score if the category has applicable audits, or no audits whatsoever.
+    if (category.auditRefs.length === 0 || this.hasApplicableAudits(category)) {
```

Comment:
can you move the length check inside `hasApplicableAudits`? it logically makes sense in there. 0 audits shouldn't be considered "has applicable audits"

Reply:
> it logically makes sense in there. 0 audits shouldn't be considered "has applicable audits"

that's why I left it out - 0 audits isn't considered "has applicable audits" so we shouldn't return true

Reply:
I'm saying it should return false in that case; `if (category.auditRefs.length === 0) return false.`

Reply:
ohh ok

Reply:
wait but we want it to render the numerical score if it has 0 audits

Reply:
we do? I don't have any context and am just jumping in out of surprise :) but if someone creates a category without any audits I would expect a gray "not applicable" rather than a red 0

Reply:
i dont think a category of 0 audits make sense? the category would get filtered out by the runner (try `--only-audits=font-size` for ex).

the only reason the demo "now" pages have it category with 0 audits is because the plugin category data is nonsensical.

Reply:
I agree it doesn't make sense, but I can imagine a company makes a custom config where such a scenario happens. I'm confused though @connorjclark I thought you just wanted this function to handle the 0 audit case are you saying now we should ignore what happens when there are 0 audits?

Reply:
nvm. logic is hard.